### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by end-4**
# Fixes
- Workspace animation fix: left-to-right and right-to-left instead of just left-to-right (and enable it by default)
- Fixed resize speed of y dimension (for tiled windows) 
# Added features
- Adjustable `splitratio` dispatcher's speed (for example `+0.1` instead of `+`)
- Added a separate value for resize animation speed (for tiled windows)

(also sorry for that messy commit history... just check the files)

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -51,10 +51,11 @@ void ConfigManager::init() {
 
     // animations
     configValues["animations:speed"].floatValue = 1;
+    configValues["animations:windowresizespeed"].floatValue = 1;
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:54]: can this be in snake_case to keep unity with the rest of the config plox

File: src/windowManager.cpp [hunk 2]
```diff
@@ -2557,16 +2584,14 @@ void CWindowManager::changeSplitRatioCurrent(const char& dir) {
...
-        default:
-            Debug::log(ERR, "changeSplitRatioCurrent called with an invalid dir!");
-            return;
+    if (dir == "+") {
+        PARENT->setSplitRatio(PARENT->getSplitRatio() + 0.05f);
+    }
+    else if (dir == "-") {
+        PARENT->setSplitRatio(PARENT->getSplitRatio() - 0.05f);
+    }
+    else {
```
Comment (vaxerski) [inline src/windowManager.cpp:2593]: {} not needed

File: src/windowManager.cpp [hunk 3]
```diff
@@ -2481,21 +2487,42 @@ void CWindowManager::recalcAllDocks() {
...
+                if (ConfigManager::getInt("animations:workspaces") == 1)
+                    work.setCurrentOffset(Vector2D(-PMONITOR->vecSize.x, 0));
+                else
+                    work.setCurrentOffset(Vector2D(0, 0));
+                work.setGoalOffset(Vector2D(0, 0));
+                work.setAnimationInProgress(true);
+            }
+        }
+    }
+    else {  // Wipe from right to left (oldwork < newwork)
```
Comment (vaxerski) [inline src/windowManager.cpp:2509]: } else {

**Review by vaxerski — CHANGES_REQUESTED** [review]
and in general if you could also make a tight list of changes needed to the wiki as iirc I have to edit it manually (cuz gh wiki)

File: src/config/ConfigManager.cpp (hunk 1 above)
Comment (end-4) [reply src/config/ConfigManager.cpp]: https://github.com/end-4/hyprland-is-great-but-i-want-xorg/commit/0bcb72e3c3476c23d08f0f127e50a1b594f035bb

File: src/windowManager.cpp (hunk 2 above)
Comment (end-4) [reply src/windowManager.cpp]: https://github.com/end-4/hyprland-is-great-but-i-want-xorg/commit/0bcb72e3c3476c23d08f0f127e50a1b594f035bb

File: src/windowManager.cpp (hunk 3 above)
Comment (end-4) [reply src/windowManager.cpp]: https://github.com/end-4/hyprland-is-great-but-i-want-xorg/commit/0bcb72e3c3476c23d08f0f127e50a1b594f035bb

File: src/utilities/AnimationUtil.cpp [hunk 4]
```diff
@@ -8,6 +8,7 @@ void AnimationUtil::move() {
     lastFrame = std::chrono::high_resolution_clock::now();
 
     const double ANIMATIONSPEED = std::max(1.f / ((double)ConfigManager::getFloat("animations:speed") * DELTA) * 462.f, (double)1.f);
+    const double RESIZEANIMATIONSPEED = std::max(1.f / ((double)ConfigManager::getFloat("animations:windowresizespeed") * DELTA) * 462.f, (double)1.f);
```
Comment (vaxerski) [inline src/utilities/AnimationUtil.cpp:11]: forgot to use here the new name

**Review by vaxerski — CHANGES_REQUESTED** [review]
all good except for this mistake

File: src/utilities/AnimationUtil.cpp (hunk 4 above)
Comment (end-4) [reply src/utilities/AnimationUtil.cpp]: https://github.com/hyprwm/Hypr/commit/c71f05473ee0c04e64496df019dd6542a60c474a

**Review by vaxerski — APPROVED** [review]

**Comment by vaxerski** [comment]
thanks!

**Pull request merged**